docs/reference/search/validate.asciidoc

```diff
@@ -163,7 +163,7 @@ GET twitter/tweet/_validate/query?rewrite=true
 }
 --------------------------------------------------
 // CONSOLE
-// TEST[skip:https://github.com/elastic/elasticsearch/issues/18254]
+// TEST[skip:the output is randomized depending on which shard we hit]
```

Comment:
Maybe used `all_shards=true` here too? That way people can start getting used to those parameters being paired together. Then you don't need a new example for it, just a paragraph about how if you leave it off then the responses are unpredictable because of the shard randomization.